app/jobs/resolution_proofing_job.rb

```diff
@@ -115,8 +113,7 @@ def proof_lexisnexis_ddp_with_threatmetrix_if_needed(
     ddp_pii = applicant_pii.dup
     ddp_pii[:threatmetrix_session_id] = threatmetrix_session_id
     ddp_pii[:email] = user&.confirmed_email_addresses&.first&.email
-    ddp_pii[:input_ip_address] = request_ip
-    ddp_pii[:local_attrib_1] = uuid_prefix
+    ddp_pii[:request_ip] = request_ip
```

Comment:
the DDP proofer reads `:uuid_prefix` and `:request_ip` from the applicant:

https://github.com/18F/identity-idp/blob/85a46ff620b38bc6870d5641f64530128c60ad73/app/services/proofing/lexis_nexis/ddp/verification_request.rb#L30-L31